wallet/api/api.go

```diff
@@ -126,6 +126,13 @@ type Interactor interface {
 	// It returns true if the user approved the signing of the transaction,
 	// false otherwise.
 	RequestTransactionReviewForSigning(ctx context.Context, traceID, hostname, wallet, pubKey, transaction string, receivedAt time.Time) (bool, error)
+
+	// RequestTransactionReviewForChecking is used to trigger a user check of the
+	// transaction a third-party application wants to receive. The wallet doesn't
+	// check the transaction.
+	// It returns true if the user approved the checking of the transaction,
+	// false otherwise.
```

Comment:
```suggestion
	// RequestTransactionReviewForChecking is used to prompt the user to check the
	// transaction a third-party application wants to receive. The wallet doesn't
	// check the transaction.
	// It returns true if the user approved the transaction,
	// false otherwise.
```

Reply:
This has been fixed. Thanks!